30 paskaita (egzaminas)/Task 5/script.js

```diff
@@ -19,3 +19,15 @@ const users = [
   { id: '8', name: 'Simon Peterson', age: 30, hasDog: false },
   { id: '9', name: 'Daniel Cane', age: 51, hasDog: true },
 ];
+
+function filterDogOwers () {
+  const newData = users.filter(element => element.hasDog === true)
+  console.log(newData)
```

Comment:
Viskas gerai, bet reikatu, kad si funkcija grazintu "users", kurie turi augintinius.

```suggestion
  return users.filter(element => element.hasDog === true)
```